odl_video/settings.py

```diff
@@ -110,7 +141,7 @@ def get_var(name, default):
 
 if get_var('USE_SHIBBOLETH', '') == 'True':
```

Comment:
This should be `get_var('USE_SHIBBOLETH', False) is True`. `get_var` parses the environment variable as a Python expression so `get_var` will never return `'True'` unless you explicitly quote it